src/imp/platform/node/platform_node.js

```diff
@@ -55,11 +54,11 @@ export default class PlatformNode {
                 parseInt(requiredMatch[2], 10),
             ];
             if (actual[0] > required[0]) {
-                return;
+                // eslint-disable-next-line no-empty
```

Comment:
removing the return statement and replacing it with a no-empty lint comment seems a bit weird to me. 
were the returns removed because of the `consistent-return` rule? perhaps cleaner to just return undefined in the bottom of the function.

Reply:
You're right that it's weird. This block should probably be rewritten.

But what's weirder, is that the removal of `return` is due to `eslint --fix`! So I vote to leave it as is in this PR.